File: Java/Activities/Car.java

public class Car {
	
	String color;
	String transmission;
	int make;
	int tyres;
	int doors;
	
	Car(){
		this.tyres = 4;
		this.doors = 4;
	}
	
	public void displayCharacteristics() {
		System.out.println("Color of the car is " + color);
		System.out.println("Transmission of the car is " + transmission);
		System.out.println("Make of the car is " + make);
		System.out.println("No. of Tyres on the car are " + tyres);
		System.out.println("No. of doors on the car are " + doors);
	}
	
	public void accelarate() {
		System.out.println("Car is moving forward.");
	}
	
	public void brake() {
		System.out.println("Car has stopped.");
	}
	 

}

File: Python/Activities/Car.py
class Car:

    def __init__(self, manufacturer, model, make, transmission, color):
        self.manufacturer = manufacturer
        self.model = model
        self.make = make
        self.transmission = transmission
        self.color = color

    def accelerate(self):
        print(self.manufacturer+ " "+ self.model+ " is moving.")

    def stop(self):
        print(self.manufacturer + " "+self.model + " has stopped.")

c1 = Car("Honda", "H1", "2012", "Manual", "blue")
c1.accelerate()
c1.stop()
c2 = Car("Hyundai", "H2", "2018", "automatic", "black")
c2.accelerate()
c2.stop()
c3 = Car("Maruti", "H5", "2022", "Manual", "Red")
c3.accelerate()
c3.stop()